app/modules/media/media_controller.py

```
@@ -224,6 +297,143 @@ async def get_message_attachments(self, message_id: str) -> List[AttachmentInfo]
                 status_code=500, detail="Failed to get message attachments"
             )
 
+    async def validate_document_upload(
+        self,
+        conversation_id: str,
+        file_size: int,
+        file_name: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `file_name`

(ARG002)